lms/static/scripts/frontend_apps/components/dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ArrowLeftIcon,
+  CaretRightIcon,
+  Link,
+} from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { Link as RouterLink } from 'wouter-preact';
+
+type BreadcrumbLink = {
+  title: string;
+  href: string;
+};
+
+export type DashboardBreadcrumbsProps = {
+  links: BreadcrumbLink[];
+};
+
+function BreadcrumbLink({ title, href }: BreadcrumbLink) {
+  return (
+    <RouterLink href={href} asChild>
+      <Link underline="hover" variant="text-light" classes="truncate">
+        <ArrowLeftIcon className="inline-block md:hidden mr-1 align-sub" />
+        {title}
+      </Link>
+    </RouterLink>
+  );
+}
+
+export default function DashboardBreadcrumbs({
+  links,
+}: DashboardBreadcrumbsProps) {
+  return (
+    <div
+      className="flex flex-row gap-0.5 w-full font-semibold"
+      data-testid="breadcrumbs-container"
+    >
+      {links.map(({ title, href }, index) => {
+        const isLastLink = index === links.length - 1;
+        return (
+          <span
+            key={`${index}${href}`}
+            className={classnames('gap-0.5', {
+              // In mobile devices, show only the last link
+              'md:flex hidden': !isLastLink,
+              'flex max-w-full': isLastLink,
+              // Distribute max width for every link as evenly as possible.
+              // Setting explicit classes here is required for Tailwind to
+              // properly detect they are being used.
```

Comment:
Can you explain this a bit more?

Reply:
Yes. Due to how Tailwind works, you can't dynamically generate class names, like this: `md:max-w-[${100 / links.length}%]`.

Something like that would not be captured at build time, and the CSS would not be included.

It is explained here https://tailwindcss.com/docs/content-configuration#dynamic-class-names, I will add a reference to this link in that comment.